### ISSUE TITLE
An option to keep original cutscenes?

### ISSUE THREAD

**Issue by tomster785** [fig02/Better-OoT#13]
I know this is intended for speedrunners, and it's mainly to make that specific experience more enjoyable by removing cumbersome cutscenes and RNG. I appreciate that, but I'm a casual player and I love what this mod does in most areas. But I still want the original story experience, that's a big draw to the game for me.

So while I know what I'm asking goes against the core philosophy of what this is meant to do, I feel an option to retain the original cutscenes would basically make this the ultimate version of OoT. I'm not sure if it's a difficult thing to do, because I honestly don't have a clue how to do this sort of stuff. If I did, I'd do it myself and not bother you with my inane request. But if it's not too much trouble to ask, I think by just making the short cutscenes toggle-able, it could make this the definitive version of one of the best games of all time. All the little gripes that I have with this game, the long chest opening, the pause menu when switching boots, the absurdly slow text, and so on are fixed, but I dunno, it just doesn't feel right without the cutscenes.

Either way, I appreciate what you're doing.

**Comment by ngsylar** [comment]
I agree. I'm very grateful for the many fixes you've made, but it would work better for me if all the cutscenes were there.

**Comment by tstambaugh92** [comment]
I'm not sure if the issues section of github is the right place for this but I agree. I'd kill for just having boots and ocarina on dpad and a few other options like disabling the owl and Dampe first try. I want a better oot experience like the name implies, but not nessesarily a stripped out one.

**Comment by fig02** [comment]
Hi
Thank you for your suggestion. 
I have no plans to implement this, because as stated in the original post, this is a tool created specifically for speedrunning. Removing all of the cutscenes is pretty much the whole point.

It probably wouldn't be too hard to implement but I have other things I want to focus my time on. Additionally I'm running out of space in the GUI for more options anyway. My apologies.

**Comment by ShadowOne333** [comment]
Hi @fig02 
I would like to implement the original cutscenes of the game into a custom fork of Better OoT.
Do you happen to know which code exactly is in charge of removing the cutscenes? So that I can restore them in the fork.

Thanks in advance!

**Comment by fig02** [comment]
The bulk of it will be in patches.py and is commented accordingly. Some of them may be applied through  assembly though

**Comment by ShadowOne333** [comment]
Thanks!
I have two questions regarding that:

1) Correct me if I'm wrong, but are the main story cutscenes the ones with the "write_bits_to_save" command starting at line 742? And up to 781 I think.
2) The ones applied through ASM are the ones inside cutscenes.asm? Or are some cutscenes in other .asm files? In the case of this, how can I go about recompiling Better OoT to add the new .asm changes? Mainly because I don't know the exact instructions to compile Better OoT from source to include any subsequent changes to .asm files.

One last thing, I am running a 32bit Gentoo distro (Linux), and I'm getting an exec error on both the Decompress and Compress binaries. I believe this is caused because the binaries are in 64bit format. How can I compile those two binaries for 32bits?

**Comment by W0lfwang** [comment]
I am also very interested in restoring the original cutscenes to the game, the ability to switch the boots on the fly, the fast text and the capacity of playing on the wii that your mod offers seriously improve the game, this is the version that I will recommend to anyone who wishes to play this game for the first time or relived it over the 3ds version, but the lack of cutscenes make this recommendation impossible. I’m not going to just ask you to do it but instead ask you, what do you need to make this a reality? what can we offer you to add the option of keeping the old cutscenes plus everything that your mod offers? 

Or if anybody knows another way of switching the boots without using the pause button and also add this to de wii, can you please tell me how?
